fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -128,9 +132,10 @@ public void testSearchQuantity_Quantity_withPrefix_GT() throws Exception {
 
     @Test
     public void testSearchQuantity_Quantity_withPrefix_LE() throws Exception {
+        // Implicit range of indexed 'Quantity' value is 24.5 to 25.5
         assertSearchDoesntReturnSavedResource("Quantity", "le24|http://unitsofmeasure.org|s");
         assertSearchDoesntReturnSavedResource("Quantity", "le24.4999||s");
-        assertSearchReturnsSavedResource("Quantity", "le24.5||s");
+        assertSearchDoesntReturnSavedResource("Quantity", "le24.5||s");
         assertSearchReturnsSavedResource("Quantity", "le25||s");
```

Comment:
Would be good to have both edges of where the resource is returned vs not returned. ("le 24.9999||s"?)

Reply:
In this case, the edges are search values of 24.5 (won't return resource) and a value greater than 24.5 (will return a resource). Already have the 24.5 case - added a case of 24.5001.